app/frontend/src/utils/transformUtils.js

```diff
@@ -1,4 +1,5 @@
 import { IdentityMode } from '@/utils/constants';
+import formioUtils from 'formiojs/utils';
```

Comment:
Style: Normally we tend to import external libraries first before we import our internal codebase. Would be good to have this line be above Line 1.

Reply:
oved this import in new commit